### ISSUE TITLE
Image Sequence Noise

### ISSUE THREAD

**Issue by bananajoe182** [zsyzzsoft/co-mod-gan#39]
Hi,
first of all thanks you for this brilliant work!

Is there a way to use this for a sequence of images with some kind of temporal consistency ? I`m looking for a way to force the generator to always sample from the same latent noise so that the outputs are always the the same. Like a predefined seed to be able to regenerate the same results. Any idea? Would this also be possible with an already pretrained model ?

Many Thanks!

**Comment by zsyzzsoft** [comment]
I think maybe not very possible without extra training...

**Comment by bananajoe182** [comment]
Ok, thanks. But if I train a custom dataset any idea where I would need to modify the model to generate images from the same latent noise and not randomly ?

**Comment by zsyzzsoft** [comment]
With the pre-trained model, you can also run inference using the same latent noise by making [this line]() a constant. However, my concern is that even if the latent noise is kept unchanged, the model still does not guarantee temporal consistency without extra training, but maybe you can have a try.

**Comment by bananajoe182** [comment]
@zsyzzsoft  Thank you very much, I will give it a try.
